src/aws/osml/model_runner/app.py

```diff
@@ -758,10 +758,69 @@ def aggregate_features(
         if image_request_item.tile_size and image_request_item.tile_overlap:
             # Read all the features from DDB.
             features = feature_table.get_features(image_request_item.image_id)
+            logger.info(f"Total features aggregated: {len(features)}")
         else:
             raise AggregateFeaturesException("Tile size and overlap must be provided for feature aggregation")
         return features
 
+    @staticmethod
+    def identify_overlap(
```

Comment:
Is there a unit test for this?

Reply:
Will follow up w/ a tasks to move this logic and implement specific unit testing.